grafana-pyroscope.yaml

```diff
@@ -0,0 +1,96 @@
+package:
+  name: grafana-pyroscope
+  version: 1.13.1
+  epoch: 0
+  description: Continuous Profiling Platform. Debug performance issues down to a single line of code
+  copyright:
+    - license: AGPL-3.0-only
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - cairo-dev
+      - go
+      - node-gyp
+      - nodejs-20
+      - pango-dev
+      - python3
+      - yarn
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 4797950b45e3fbf1f44e704819cd1dfd81b78f0c
+      repository: https://github.com/grafana/pyroscope
+      tag: v${{package.version}}
+
+  - runs: |
+      # https://github.com/grafana/pyroscope/blob/3da96b8e449de267d4663e14207b8b272f9edc6d/.github/workflows/release.yml#L48C14-L48C33
+      mkdir -p pyroscope/scripts/webpack
+      mkdir -p pyroscope/public/app
+      mkdir -p pyroscope/public/templates
+      cp yarn.lock package.json tsconfig.json ./pyroscope
+      cd pyroscope
+      yarn --frozen-lockfile
+      cp -r ../scripts/webpack ./scripts/
+      cp -r ../public/app ./public/
+      cp -r ../public/templates ./public/
+      yarn build
+      cp -r ./public/build ../public/build
```

Comment:
Should frontend be sub-package instead? I couldn't see frontend related files in the final APK: https://apk.chaindag.dev/size/apk.cgr.dev/wolfi-presubmit/c094e8f89181e977cca15915a9c60867e2a8780b/x86_64/grafana-pyroscope-1.13.1-r0.apk@etag:39336462613236373130396439653033336233636136643036366131353032382d39

Reply:
given frontend is sensitive to this package, can we do some html querying if possible if they expose some endpoint. (basic grep for html works as well)

Reply:
I tried cypress tests but its hard to make it work due to dependencies so I'll do the cypress tests at image level using docker so I just added simple ui access tests.